projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -0,0 +1,139 @@
+import { getScoreLetter, requestSpeedScores } from '@automattic/jetpack-boost-score-api';
+// We'll need the IconTooltip in a follow-up PR. TODO: remove me
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { IconTooltip, Spinner, BoostScoreBar } from '@automattic/jetpack-components';
+import { __, sprintf } from '@wordpress/i18n';
+import React, { useCallback, useEffect, useState } from 'react';
+import useAnalytics from '../../../hooks/use-analytics';
+import './style.scss';
+
+const BoostSpeedScore = () => {
+	const { recordEvent } = useAnalytics();
+	const [ isLoading, setIsLoading ] = useState( false );
+	const [ speedLetterGrade, setSpeedLetterGrade ] = useState( 'C' );
+	const [ daysSinceTested, setDaysSinceTested ] = useState( 1 );
+	const [ averageSpeedScore, setAverageSpeedScore ] = useState( 0 );
+	const [ isSpeedScoreError, setIsSpeedScoreError ] = useState( false );
+
+	const { apiRoot, apiNonce, connectionStatus } = window.JP_CONNECTION_INITIAL_STATE;
```

Comment:
I see most other places using `useMyJetpackConnection()` to get this type of info, is there an advantage to using the window here?